app/models/smart_proxy.rb

```diff
@@ -30,7 +30,7 @@ class SmartProxy < ActiveRecord::Base
   scoped_search :on => :url, :complete_value => :true
   scoped_search :in => :features, :on => :name, :rename => :feature, :complete_value => :true
 
-  delegate :version, :tftp_server, :to => :proxy_status
+  delegate :version, :tftp_server, :environment_stats, :to => :proxy_status
```

Comment:
can you please add a TODO as this needs to move to its own proxy status class (see #3069)